LEAF_Request_Portal/admin/header-test.php

```diff
@@ -20,7 +20,8 @@
 </section>
 
 <header id="header" class="usa-header site-header">
-    <div class="usa-navbar site-header-navbar">
+    <nav class="usa-navbar site-header-navbar">
+        
         <div class="usa-logo site-logo" id="logo">
             <em class="usa-logo__text">
                 <a onclick="window.location='./'" title="Home" aria-label="LEAF home" class="leaf-cursor-pointer">
```

Comment:
Refrain from using "window.location" it imposes security concerns

Reply:
Ok removed legacy code and replaced with href